extensions/extensions-docker/aissemble-spark-infrastructure/src/main/resources/docker/Dockerfile

```diff
@@ -4,8 +4,9 @@ FROM ${DOCKER_BASELINE_REPO_ID}boozallen/aissemble-spark:${VERSION_AISSEMBLE}
 
 LABEL org.opencontainers.image.source = "https://github.com/boozallen/aissemble"
 
+RUN curl -L https://github.com/delta-io/connectors/releases/download/v0.6.0/delta-hive-assembly_2.12-0.6.0.jar \
+          -o "${SPARK_HOME}"/jars/delta-hive-assembly_2.12-0.6.0.jar
```

Comment:
A: Should probably pass the connector version as a docker build arg from the POM so we can drive version numbers from Maven.

Reply:
Agree

- [ ]  Move version to build arg